example/android/app/src/main/java/com/klaviyoreactnativesdkexample/MainActivity.kt

```diff
@@ -17,4 +32,113 @@ class MainActivity : ReactActivity() {
    * which allows you to enable New Architecture with a single boolean flags [fabricEnabled]
    */
   override fun createReactActivityDelegate(): ReactActivityDelegate = DefaultReactActivityDelegate(this, mainComponentName, fabricEnabled)
+
+  /**
+   * Launches a permission request, and receives the result in the callback below
+   */
+  private var requestPermissionLauncher: ActivityResultLauncher<String> =
```

Comment:
There's no way in RN to do this right?

Reply:
Yeah there is, there are 3rd party libs for cross platform permissions. I think right now we mention them in the readme but don't implement them in the sample app

Reply:
Oh I thought I saw something in the .74 release but it was unrelated to push.

Reply:
this is just a generic permission request launcher, works roughly the same regardless of permission type (with some exceptions, like geolocation permissions have additional tiers)